src/plugins/irc-events/names.js

```diff
@@ -10,7 +10,12 @@ module.exports = function(irc, network) {
 			return;
 		}
 
-		chan.users = data.users.map(u => new User(u, network.prefixLookup));
+		chan.users = data.users.map(user => {
```

Comment:
Hostmasks might be useful for automated banning, etc. I guess no point in keeping it in if we don't use it just now, though.